### ISSUE TITLE
Why did you deprecate conversion of frozen graphs??? That is something people still use!

### ISSUE THREAD

**Issue by Tylersuard** [tensorflow/tfjs#3501]
Also, what in the world is an output node?  I am required to know my output node names for converting my frozen graph, but I have no clue where to find them.

#### TensorFlow.js version
0.8.6

#### Describe the problem or feature request

The documentation describes the process for conversion from frozen graph to tensorflow.js model as follows:

tensorflowjs_converter \
    --input_format=tf_frozen_model \
    --output_node_names='MobilenetV1/Predictions/Reshape_1' \
    /mobilenet/frozen_model.pb \
    /mobilenet/web_model

However, I am not using Mobilenet.  I am using my own custom trained model.

Can someone please help me convert my .pb frozen model to a tensorflow.js model?

**Comment by Tylersuard** [comment]
For those of you searching for this same issue, you can find the output nodes in the .py file that created the frozen graph.  For me, the output nodes were:

and the following output nodes returned by the model.postprocess(..):
  * `num_detections`: Outputs float32 tensors of the form [batch]
      that specifies the number of valid boxes per image in the batch.
  * `detection_boxes`: Outputs float32 tensors of the form
      [batch, num_boxes, 4] containing detected boxes.
  * `detection_scores`: Outputs float32 tensors of the form
      [batch, num_boxes] containing class scores for the detections.
  * `detection_classes`: Outputs float32 tensors of the form
      [batch, num_boxes] containing classes for the detections.
  * `raw_detection_boxes`: Outputs float32 tensors of the form
      [batch, raw_num_boxes, 4] containing detection boxes without
      post-processing.
  * `raw_detection_scores`: Outputs float32 tensors of the form
      [batch, raw_num_boxes, num_classes_with_background] containing class score
      logits for raw detection boxes.
  * `detection_masks`: (Optional) Outputs float32 tensors of the form
      [batch, num_boxes, mask_height, mask_width] containing predicted instance
      masks for each box if its present in the dictionary of postprocessed
      tensors returned by the model.
  * detection_multiclass_scores: (Optional) Outputs float32 tensor of shape
      [batch, num_boxes, num_classes_with_background] for containing class
      score distribution for detected boxes including background if any.
  * detection_features: (Optional) float32 tensor of shape
      [batch, num_boxes, roi_height, roi_width, depth]
  containing classifier features
